libobs/audio-monitoring/win32/wasapi-output.c

```diff
@@ -28,6 +28,8 @@ ACTUALLY_DEFINE_GUID(IID_IAudioClient, 0x1CB9AD4C, 0xDBFA, 0x4C32, 0xB1, 0x78,
 ACTUALLY_DEFINE_GUID(IID_IAudioRenderClient, 0xF294ACFC, 0x3146, 0x4483, 0xA7,
 		     0xBF, 0xAD, 0xDC, 0xA7, 0xC2, 0x60, 0xE2);
 
+volatile long audio_track_active;
```

Comment:
This variable should be static.
```suggestion
static volatile long audio_track_active;
```